web.tf

```diff
@@ -15,19 +15,19 @@ resource "aws_security_group" "acme_web" {
     from_port   = 22
     to_port     = 22
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0"]
```

Comment:
**Accurics remediation**
 
**[HIGH]: http port open to internet**
 
**Vulnerability**
http port open to internet
 
**Remediation**
Change cidr_block to non 0.0.0.0/0